.github/workflows/c-cpp.yml

```diff
@@ -0,0 +1,32 @@
+name: C/C++ CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: update
+      run: sudo apt update
+    - name: install packages
+      run: sudo apt-get install --no-install-recommends -y debhelper-compat devscripts libapt-pkg-dev gettext libgtk-3-dev libvte-2.91-dev intltool xmlto sharutils libept-dev
+      # libsm-dev lsb-release (already installed on image)
+    - name: autogen.sh
+      run: ./autogen.sh
+    - name: configure
+      run: ./configure
+    - name: make
+      run: make
+    - name: debuild
+      run: debuild -B -us -uc
+#    - name: make check
```

Comment:
I guess we could try installing it too (and removing it) to check for postinst/postrm issues.